### PR TITLE
Improve StorIOSQLite Javadocs

### PR DESCRIPTION
**Pull request by artem-zinnatullin**
@nikitin-da PTAL

### CODE REVIEW

File: storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/operation/delete/PreparedDeleteByQuery.java [hunk 1]
```diff
@@ -42,14 +43,23 @@ public DeleteResult executeAsBlocking() {
     }
 
     /**
-     * Creates an {@link Observable} which will emit result of Delete Operation
+     * Creates {@link Observable} which will perform Delete Operation and send result to observer.
+     * <p>
+     * Returned {@link Observable} will be Cold Observable, which means that it performs
```
Comment (nikitin-da) [inline storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/operation/delete/PreparedDeleteByQuery.java]: Quotes?!

File: storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/operation/delete/PreparedDeleteObject.java [hunk 2]
```diff
@@ -45,14 +45,23 @@ public DeleteResult executeAsBlocking() {
     }
 
     /**
-     * Creates an {@link Observable} which will emit result of Delete Operation
+     * Creates {@link Observable} which will perform Delete Operation and send result to observer.
+     * <p>
+     * Returned {@link Observable} will be Cold Observable, which means that it performs
```
Comment (nikitin-da) [inline storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/operation/delete/PreparedDeleteObject.java]: Quotes?!

File: storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/operation/delete/PreparedDeleteObjects.java [hunk 3]
```diff
@@ -88,9 +93,18 @@
     }
 
     /**
-     * Creates an {@link Observable} which will emit results of Delete Operation
+     * Creates {@link Observable} which will perform Delete Operation and send result to observer.
+     * <p>
+     * Returned {@link Observable} will be Cold Observable, which means that it performs
```
Comment (nikitin-da) [inline storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/operation/delete/PreparedDeleteObjects.java]: Yet another quotes

File: storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/operation/exec_sql/PreparedExecSql.java [hunk 4]
```diff
@@ -25,17 +29,37 @@
...
     public Void executeAsBlocking() {
         storIOSQLite.internal().execSql(rawQuery);
         return null;
     }
 
+    /**
+     * Creates {@link Observable} which will perform Execute SQL Operation
+     * and send result to observer.
+     * <p>
+     * Returned {@link Observable} will be Cold Observable, which means that it performs
```
Comment (nikitin-da) [inline storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/operation/exec_sql/PreparedExecSql.java]: I will mark all places when type of `Observable` hasn't quotes

File: storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/operation/delete/PreparedDeleteByQuery.java (hunk 1 above)
Comment (artem-zinnatullin) [inline storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/operation/delete/PreparedDeleteByQuery.java]: Okay :)

File: storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/operation/put/PreparedPutContentValues.java [hunk 5]
```diff
@@ -36,19 +39,28 @@ public PutResult executeAsBlocking() {
     }
 
     /**
-     * Creates {@link Observable} which will perform Put Operation and send result to observer
+     * Creates {@link Observable} which will perform Put Operation and send result to observer.
+     * <p>
+     * Returned {@link Observable} will be Cold Observable, which means that it performs
```
Comment (nikitin-da) [inline storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/operation/put/PreparedPutContentValues.java]: So cold=)

File: storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/operation/put/PreparedPutContentValuesIterable.java [hunk 6]
```diff
@@ -120,7 +120,7 @@
          *
          * @param putResolver put resolver
          * @return builder
-         * @see {@link DefaultPutResolver} — easy way to create {@link PutResolver}
+         * @see DefaultPutResolver
```
Comment (nikitin-da) [inline storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/operation/put/PreparedPutContentValuesIterable.java]: Without `@link`?

File: storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/operation/put/PreparedPutObject.java [hunk 7]
```diff
@@ -37,9 +42,18 @@ public PutResult executeAsBlocking() {
     }
 
     /**
-     * Creates {@link Observable} which will perform Put Operation and send result to observer
+     * Creates {@link Observable} which will perform Put Operation and send result to observer.
+     * <p>
+     * Returned {@link Observable} will be Cold Observable, which means that it performs
```
Comment (nikitin-da) [inline storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/operation/put/PreparedPutObject.java]: Quotes?!

File: storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/query/InsertQuery.java [hunk 8]
```diff
@@ -115,17 +118,18 @@ public CompleteBuilder table(@NonNull String table) {
...
-         * Optional: Specifies null column hack
+         * Optional: Specifies {@code NULL} column hack.
          * <p/>
          * SQL doesn't allow inserting a completely empty row without naming at least one column name.
          * If your provided values are empty, no column names are known and an empty row can't be inserted.
          * If not set to null, the nullColumnHack parameter provides the name of nullable column name
          * to explicitly insert a NULL into in the case where your values is empty.
          * <p/>
-         * Default value is <code>null</code>
+         * Default value is {@code null}.
```
Comment (nikitin-da) [inline storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/query/InsertQuery.java]: Shell `null` be capital?

Comment (artem-zinnatullin) [inline storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/query/InsertQuery.java]: No

**Comment by artem-zinnatullin** [comment]
Rebased onto master and added Quotes to "Hot" and "Cold" :)

File: storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/operation/put/PreparedPutContentValuesIterable.java (hunk 6 above)
Comment (artem-zinnatullin) [inline storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/operation/put/PreparedPutContentValuesIterable.java]: Yes, when you use `@see` -> `@link` placed automatically and if you'll write `@see {@link ...}` link will be broken!

Comment (nikitin-da) [inline storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/operation/put/PreparedPutContentValuesIterable.java]: Oh, thanks

**Comment by nikitin-da** [comment]
Cool fixes!

**Pull request merged**